### ISSUE TITLE
No love from SCRIPT=travis-verify.sh

### ISSUE THREAD

**Issue by zielint0** [INRIA/spoon#2453]
Am I doing something wrong or it is a repo access issue?

```
[WARNING] The repository url 'https://repository.apache.org/releases/' is invalid - Repository 'sonatype-apache' will be blacklisted.
[ERROR] Unable to determine if resource com.fasterxml.jackson.core:jackson-annotations:jar:2.9.0:compile exists in http://download.java.net/maven/2/
[ERROR] Unable to determine if resource com.fasterxml.jackson.core:jackson-core:jar:2.9.2:compile exists in http://download.java.net/maven/2/
[ERROR] Unable to determine if resource com.fasterxml.jackson.core:jackson-databind:jar:2.9.2:compile exists in http://download.java.net/maven/2/
[ERROR] Unable to determine if resource com.github.ste
```

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:30 min
[INFO] Finished at: 2018-09-06T21:49:15Z
[INFO] Final Memory: 57M/191M
[INFO] ------------------------------------------------------------------------
Traceback (most recent call last):
  File "./chore/check-links-in-doc.py", line 5, in <module>
    import CommonMark
ImportError: No module named CommonMark
The command "./chore/travis/$SCRIPT" exited with 1.
```

**Comment by monperrus** [comment]
fixed by #2458

**Comment by zielint0** [comment]
@monperrus 
Thank you sooo much! :1st_place_medal:
